lib/i18n/traditional-chinese.js

```diff
@@ -0,0 +1,107 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Genie
+//
+// Copyright 2018-2019 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Giovanni Campagna <gcampagn@cs.stanford.edu>, Johnny Hsu <johnny.chhsu01@gmail.com>
+//
+// See COPYING for details
+"use strict";
+
+function replaceMeMy(sentence) {
+    sentence = sentence.replace(/\b((?!(?:通知|告知)\b)[a-zA-Z0-9]+) me\b/g, '$1 他們');
```

Comment:
I tried to dig into git history to figure out what this line is doing but I can't quite understand it. I wonder if we can drop it.
In any case, `[a-zA-Z0-9]+` is a character class that matches a single word in English, but it won't do in Chinese, where characters are not a-z. If you want to replicate it exactly, you'll need to use the CJK ideograph range, possibly as a Unicode character class (and adding the `u` flag to the regexp).
Also, you'll probably want 我 instead of `me`?

Also, here and later, English uses `their` as gender neutral *singular* pronoun (because it's not easy to find out the gender of the replaced username at this stage), but won't do for Chinese. Maybe just use 他 and forget about it?

Reply:
Maybe we can drop this here temporary, try some templates and see what we need?
In Chinese, there are 他 and 她, but yeah it's not a big deal in Chinese, we can use 他.